src/components/managerContent/menuContent/menuTable/table.js

```diff
@@ -0,0 +1,76 @@
+import './table.css';
+import table from './table.hbs';
+import template from './menuTable.hbs';
+import weekTab from './weektab.hbs';
+import menuItem from '../menuItem/menuItem.hbs';
+import { createElementsFromString } from '../../../../common/utils';
+import { getMenu, setMenu } from '../../../../common/menuService';
+import menuObject from '../../../../common/menuObject';
+
+export default class MenuTable {
+  render(target) {
+    // get req
+    setMenu(menuObject);
+    const weeksMenu = getMenu();
+    this.renderContent(target, weeksMenu);
+    return target;
+  }
+  renderContent(target, weeksMenu) {
```

Comment:
Please use an empty line to separate methods. This will make your code more readable.